submissions/ALT+F4 LTN-RLD-Cybersyn Mod Comparison/Requesting a Train.md

```diff
@@ -0,0 +1,111 @@
+
+{% include image.html src='Cybersyn-GUI-2.png' alt='Cybernetic Combinator in Station Mode' caption='Cybernetic Combinator in station mode' %}
+
+One of the best changes in my opinion, is the fact that Cybersyn's combinator GUI allows the user to not just notice that a Station can both Provide and Request, but also disable one of those modes. This means that having the workaround of sending very high threshold signals to the train stop is unneeded. Another excellent change is that the Combinator is a single entity that makes it more obvious where the input and output ports are. 
+
+Adding a second combinator in Station control mode additionally allows the user to set thresholds on a per-item basis. While this could be acheived with LTN, it also required additional combinators for each resource with a different threshold. With Cybersyn, the same could be done with just an additional Cybernetic combinator and a second constant combinator, making it much more space efficient for Stations with many items.
+
+In terms of downsides, there few and far between: The Cybernetic Combinator can take up a bit more space in certain circumstances. They also require power to function properly. Lastly, the automatic allow list can finicky, and can be incompatible with things like diagonal stations or modded wagons with different sizes. It also leaves no method to manually control the train configurations allowed to use the station.
+
+### Rail Logistics Dispatcher - Inbuilt GUI
+
+The second of the three mods to be released, Rail Logistics Dispatcher is also the most radically different, with its most prominent feature being an integrated GUI.
+
+{% include compare.html old='RLD=stop.png' new='RLD-stop-ALT.png' caption='The Tititular Rail Logistics Dispatcher. While it resembles a lamp, a look in ALT mode reveals the truth of the matter: It is actually a combinator in disguise!' %}
+
+Much like Cybersyn, Rail Logistics Dispatcher forgoes replacing the vanilla Train Stop in favor of adding another entity to control it. However, the Dispatcher it adds is smaller, doesn't require power, and uses lamp graphics similar to LTN, making it easier to tell whether the stop is functioning correctly. Taken together, the minimum footprint for the dispatcher and train stop is 2x2 + 1x1, with the latter needing to be 1 tile away from the stop it controls.
+
+{% include image.html src='RLD-GUI.png' alt='Rail Logistics Dispather's GUI' caption='The GUI used to configure the Dispatcher' %}
+
+While add-on mods like [LTN Combinator Modernized](https://mods.factorio.com/mod/LTN_Combinator_Modernized) have been very helpful to those lacking experience with circuit network, the fact remains that it is an entirely separate mod dedicated to simplfying and improving the user experience. With that in mind, RLD opted to include a GUI right out of the box. (RLD does also offer a 'signals mode' to allow the user to configure the Dispatcher in a manner more similar to LTN if desired.)
```

Comment:
Pretty sure Cybersyn is currently working on adding a GUI to manage the trains, which I think might be something like: https://mods.factorio.com/mod/LtnManager

Reply:
Yes, from what I know, both Cybersyn and Rail Logisitcs Dispatcher are working on their own train managing GUIs. I felt it was better to look at "current" features rather than "coming soon/promised" ones, but I can mention that for both.

Reply:
I think it's only fair to the mod author to at least mention it. If I was the mod author I wouldn't like it if someone used it as as disadvantage.